analysis/parser/RawTextParser/src/main/kotlin/de/maibornwolff/codecharta/parser/rawtextparser/model/FileMetrics.kt

```diff
@@ -0,0 +1,10 @@
+package de.maibornwolff.codecharta.parser.rawtextparser.model
+
+class FileMetrics {
+    val metricMap = mutableMapOf<String, Double>()
+
+    fun addMetric(name: String, value: Int): FileMetrics {
```

Comment:
Why is the map of type double when you can only add ints to it and convert them to double later?

Reply:
If I understand this correctly this is due to indentationLevel being an integer 100% of the time and we want to export doubles in the cc.json.
However, if we decide to add another metric that cna be a double, you'd have to change the method signature. I'd suggest to change the method signature to `value: Double`

Reply:
You are right, it is strange to only provide addMetric with Int, when in fact fileMetrics contain double values. I will solve this by providing an addMetric(name: String, value: Double) function in addition to the current one.